Makefile

```diff
@@ -15,8 +15,6 @@ PKGNAME := yay
 BINNAME := yay
 PACKAGE := ${PKGNAME}_${VERSION}_${ARCH}
 
-export GOPATH=$(shell pwd)/.go
```

Comment:
Nevermind, tried it and things appear to work… I probably just don't understand the build.

Reply:
`-mod=vendor` is the replacement for setting the gopath. The flag seems to work regardless of the `GO111MODULE` variable.

`-mod` is new to 1.11 but we target arch so there's no need to support older versions.

Reply:
I think setting -mod=vendor will now break things when `GO111MODULE=off` though, you'll get an error that the flag has no effect unless modules are being used.

Reply:
Yeah you're right. I typo'd it and did `GO111MODULES=off` in testing :/

Reply:
This leads to a problem though. When using modules, go downloads all the stuff into gopath/pkg, ignoring the vendored stuff.

`-mod=vendor` Fixes this but then breaks the build when building without modules,

Reply:
> This leads to a problem though. When using modules, go downloads all the stuff into gopath/pkg, ignoring the vendored stuff.

Is that a problem? If you set GOPATH to somewhere local that won't pollute the users GOPATH and it downloads there, you've still got `go.sum` making sure that things haven't changed.

Reply:
Maybe not so much of a problem then. It is still annoying that it redownloads the packages instead of using the ones on disk. Plus as far as pkgbuilds are concerned.

I guess I'll still merge it. The title is "Fix build" not "Support" anyway. I'll call this a work around until we merge #705.

Reply:
Ah, but then go.sum isn't checked in yet, sorry, I was forgetting about that. Yah, it is a tricky one.

Reply:
Part of the output is `go: copying requirements from Gopkg.lock` so go.sum should end up being exactly the same every time hopefully.

Reply:
A bit late, but alternatively if you want to make things easy on yourself, it just occurred to me that you could also manually enable or disable GO111MODULE in the makefile and force building a single way when Go 1.11 is used.